### DISCUSSION TITLE
Database European Components?

### DISCUSSION THREAD

**Discussion by Kaesekopf** [pvlib/pvlib-python#1390] [Q&A]
The CEC, Sandia and ASD databases for modules and inverters only seem to contain data for components for the US market.  For modules this will probably not make much of a difference, but inverters are adapted to the local grid specifications (voltage, frequency) and some inverter brands are only sold on their home market. So my question is: is anybody aware of a database with European components that is compatible with any of the pvlib models?

**Comment by campanelli-sunpower** [comment]
@Kaesekopf  I recall recently seeing 208V and 240V versions of several inverters in a CEC list, and SunPower typically incorporates both into our internal database, which is based on the Sandia Inverter Performance Model. My understanding is that the 208V inverters are the 50Hz ones for non-US markets. Is that generally true? If so, then do you mean that these 50Hz 208V inverters are generally not being added to the DBs available via pvlib-python?

**Reply by cwhanse** [reply]
I don't think inverter AC voltage has anything to do with grid frequency. 208V/240V AC voltages result from different connections to the 3 phases of grid power. @adriesse do you know of any public source of inverter model parameters for 50Hz systems?

**Reply by campanelli-sunpower** [reply]
Thanks @cwhanse. I think I was perusing the PVsyst component DBs for AC modules at one point and came to the wrong conclusion about the inverter frequency correlation, probably because that software is based out of Europe. I'm trying to get PVsyst running again in my Windows VM so that I can verify my mistake.

**Reply by adriesse** [reply]
I think it's a fair and relevant question/comment.  Manufacturers make region-specific versions of inverters and different specs can vary slightly between them as a result.  CEC is unlikely to list anything but the US versions.  I suspect that there are some open European or international inverter lists, but don't have a link handy.  Voltage can be a clue for region, but usually you need to know more.

**Reply by Kaesekopf** [reply]
An example:  I called the CEC database with:


Inverterdatabase=pvlib.pvsystem.retrieve_sam(name='CECInverter', path=None)
Just take a random European inverter SMA Sunnyboy 5000 TL  in the CEC database there are two similar entrees:

'SMA_America__SB5000TL_US_22__208V_' and 'SMA_America__SB5000TL_US_22__240V_'

European single phase Voltage is 230 V, 3 phase Voltage is 380 V. So nameplate AC Voltages already do not match.

Both have Vdcmax 480V, the German specifications* say Maximum DC Voltage is 750 V.
The US  versions in the database have Idc_max of  12 and 13A respectively, where the German one has 3 (strings) x7.5A=22.5 A

So they do appear to have different characteristics from their European counterparts.

*https://files.sma.de/downloads/SBMS-11-BD0606.pdf

**Reply by campanelli-sunpower** [reply]
My memory of PVsyst was only partially correct: Here is was the basic configuration parameters look like, which includes a selector for 50Hz and 60Hz (or both) and the (unique) phase:
![image](https://user-images.githubusercontent.com/88404760/150862679-802409bc-dc6b-4e8d-9359-3e1a48cea12b.png)

**Reply by campanelli-sunpower** [reply]
@adriesse (or others) I'm not sure how familiar with PVsyst you are, but I am wondering if "Monophased" is correct for the above illustrated 208V inverter, because my understanding now is that 208V corresponds to three-phase installations in the US.

My point here, with respect to this discussion, is should pvlib somehow track this information as well (even though it seems to be a limitation of the underlying region-centric databases and doesn't directly play a role in parameter fitting)? The PVsyst documentation isn't entirely clear, but it seems that PVsyst might use the phase choice in the computation of AC interconnection losses.

**Reply by adriesse** [reply]
@campanelli-sunpower A single-phase inverter can feed into one phase of a three-phase electrical network.

**Comment by cwhanse** [comment]
@Kaesekopf as an alternative, if you have the inverter datasheet and it includes efficiency curves at different DC voltages, you may be able to use `pvlib.inverter.fit_sandia` to get parameters for `pvlib.inverter.sandia`. But I think `pvlib.inverter.pvwatts` would produce almost the same inverter efficiency at most power levels and is much easier to use.
